apps/vmq_diversity/rebar.config

```diff
@@ -17,5 +17,5 @@
     %%Eonblast hasn't merged the Erlang 18 related PR from djustinek
     %%{emysql, {git, "git://github.com/Eonblast/Emysql.git", {tag, "v0.4.1"}}},
     {mongodb, {git, "https://github.com/comtihon/mongodb-erlang.git", {ref, "713e8bd"}}},
-    {mcd, {git, "https://github.com/EchoTeam/mcd.git", {ref, "b5b4a32"}}}
+    {mcd, {git, "https://github.com/mths1/mcd.git", {branch, "master"}}}
```

Comment:
should this be in this branch?

Reply:
I'd prefer to move mcd to vernemq repro (this is a mcd version without lager dependency)